perfdash/google-gcs-downloader.go

```diff
@@ -47,26 +47,26 @@ type GoogleGCSDownloaderOptions struct {
 	OverrideBuildCount bool
 }
 
-// GoogleGCSDownloader that gets data about Google results from the GCS repository.
-type GoogleGCSDownloader struct {
-	GoogleGCSBucketUtils *bucketUtil
-	Options              *GoogleGCSDownloaderOptions
+// Downloader that gets data about Google results from the GCS repository.
+type Downloader struct {
```

Comment:
Why is the downloader interface in the google-gcs-downloader file rather than in a generic place like downloader.go?

Reply:
So downloader.go did have a Downloader interface - but it wasn't really doing much. I want to rename these files to fit accordingly, but unfortunately, the diff would be super confusing. For instance, if I rename google-gcs-downloader.go to downloader.go, then it will show the diff b/w the old downloader.go with the renamed one (and they're two completely different files! 🙅 

We decided to rename: 
- google-gcs-downloader.go to either metrics-downloader.go or files-downloader.go 
- downloader.go to metrics-downloader-helper.go or files-downloader-helper.go

I just didn't do the renaming because the diff would be confusing.

Reply:
SGTM! +1 to MetricsDownloader